src/huggingface_hub/utils/__init__.py

```diff
@@ -46,6 +46,7 @@
 )
 from ._chunk_utils import chunk_iterable
 from ._datetime import parse_datetime
+from ._deprecation import _deprecate_method
```

Comment:
No need to import `_deprecate_method`, see the comment Line 127 in `huggingface_hub/hf_api.py`

Reply:
I had to import it, otherwise the test runs into an `ImportError: cannot import name '_deprecate_method' from 'huggingface_hub.utils'`.